bookwyrm/templates/guided_tour/home.html

```diff
@@ -0,0 +1,220 @@
+            on: "left",
+        },
+        highlightClass: 'tour-element-highlight',
+        buttons: [
+            {
+                action() {
+                    return this.back();
+                },
+                secondary: true,
+                text: "{% trans 'Back' %}",
+            },
+            {
+                action() {
+                    return this.next();
+                },
+                text: "{% trans 'Next' %}",
+            },
+        ],
+    },
+    {
+        text: "{% trans 'Books on your reading status shelves will be shown here.' %}",
+        title: "{% trans 'Your Books' %}",
+        attachTo: {
+            element: "#tour-suggested-books",
+            on: "right",
+        },
+        highlightClass: 'tour-element-highlight',
+        buttons: [
+        {
+                action() {
+                    return this.back();
+                },
+                secondary: true,
+                text: "{% trans 'Back' %}",
+            },
+            {
+                action() {
+                    return this.next();
+                },
+                text: "{% trans 'Next' %}",
+            },
+        ],
+    },
+    {
+        text: "{% trans 'Updates from people you are following will appear in your <strong>Home</strong> timeline.<br><br>The <strong>Books</strong> tab shows activity from anyone, related to your books.' %}",
+        title: "{% trans 'Timelines' %}",
+        attachTo: {
+            element: "#feed",
+            on: "left",
+        },
+        highlightClass: 'tour-element-highlight',
+        buttons: [
+        {
+                action() {
+                    return this.back();
+                },
+                secondary: true,
+                text: "{% trans 'Back' %}",
+            },
+            {
+                action() {
+                    return this.next();
+                },
+                text: "{% trans 'Next' %}",
+            },
+        ],
+    },
+    {
+        text: "{% trans 'This bell will light up when you have a new notification. When it does, click on it to find out what exciting thing has happened!' %}",
+        title: "{% trans 'Notifications' %}",
+        attachTo: {
+            element: '#tour-notifications',
+            on: "left-end",
+        },
+        buttons: [
+        {
+                action() {
+                    return this.back();
+                },
+                secondary: true,
+                text: "{% trans 'Back' %}",
+            },
+            {
+                action() {
+                    return this.next();
+                },
+                text: "{% trans 'Next' %}",
+            },
+        ],
+    },
+    {
+        text: "{% trans 'Your profile, books, direct messages, and settings can be accessed by clicking on your name here.<p class=\'notification is-warning is-light mt-3\'>Try selecting <strong>Profile</strong> from the drop down menu to continue the tour.</p>' %}",
```

Comment:
For some reason this step isn't showing for me?

Reply:
Ok I am pretty sure I have fixed this now - thanks for catching it. It was a dodgy element selector to anchor the step - see the commit message for more info.